reinforcement_learning/agents/basic_mc_agent/returns.py

```diff
@@ -81,44 +79,3 @@ def _repr_svg_(self):
 
     def view(self):
         return self._get_graph().view()
-
-
-if __name__ == '__main__':
-    # Returns test
-
-    r = Returns()
-
-    s1 = SimpleState([[-1, 0], [-1, -1]])
-    a1 = SimpleAction([1, 0])
-    G = -3
-    r[s1, a1].append(G)
-
-    s2 = SimpleState([[-1, 0], [0, -1]])
-    a2 = SimpleAction([0, 0])
-    G = 10.0
-    r[s2, a2].append(G)
-    G = 7.8
-    r[s2, a2].append(G)
-
-    s3 = SimpleState([[0, 0], [0, -1]])
-    a3 = SimpleAction([1, 1])
-    G = -5.1
-    r[s3, a3].append(G)
-    G = 3.2
-    r[s3, a3].append(G)
-    G = -1.9
-    r[s3, a3].append(G)
-
-    s4 = SimpleState([[-1, 0], [0, 0]])
-    a4 = SimpleAction([0, 0])
-    G = -5.1
-    r[s4, a4].append(G)
-    G = -6.5
-    r[s4, a4].append(G)
-    G = -9.4
-    r[s4, a4].append(G)
-    G = -10
-    r[s4, a4].append(G)
-
-    print(r)
-    # r.view()
```

Comment:
It's also necessary, because only human-eay can test if graphical appearience is good enaugh...

Reply:
It's also necessary, because only human-eay can test if graphical appearience is good enaugh...